src/act.offensive.cpp

```diff
@@ -709,8 +709,8 @@ bool _sort_pairs_by_weight(std::pair<int, int> a, std::pair<int, int> b) {
 
 ACMD(do_flee)
 {
-  if (AFF_FLAGGED(ch, AFF_PRONE)) {
-    send_to_char("It's a struggle to flee while prone!\r\n", ch);
+  if (AFF_FLAGGED(ch, AFF_PRONE) || GET_POS(ch) <= POS_SITTING) {
```

Comment:
This is already solved for in interpreter.cpp-- the FLEE command has a minimum position of POS_FIGHTING, so if you're not standing and/or fighting, you'll get "Maybe you should get on your feet first?"

Reply:
For mobs flagged MOB_WIMPY, where do_flee is called automatically at less than half health, does the interpreter apply?

Reply:
Ah, if that's what you're fixing, then this works.

Reply:
Oh, related to this, I know you and Vile use checkmarks to mark off the bug reports that are complete. Should I checkmark the ones where I send in a fix, or should I leave that to you?

Reply:
Leave it to me please, worst-case I end up forgetting it was fixed and fail to reproduce the issue. I'm looking to avoid situations where someone submits a patch and checks the issue off, but the patch doesn't make it into the codebase for whatever reason.

Reply:
Yeah, makes sense.